ddl/owner_manager.go

```diff
@@ -146,33 +151,42 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, key string, wg *sync.Wait
 	for {
 		select {
 		case <-m.etcdSession.Done():
-			// TODO: Create session again?
-			log.Warnf("[ddl] etcd session is done.")
+			log.Info("[ddl] etcd session is done, creates a new one")
+			err := m.newSession(ctx, newSessionRetryUnlimited)
+			if err != nil {
+				log.Infof("[ddl] break %s campaign loop, err %v", key, err)
```

Comment:
I will change it to warn level.
I think backoff isn't necessary because we will retry util `context` is canceled.